bin.src/compFakeCoadd.py

```diff
@@ -15,6 +18,7 @@
 
 from distutils.version import StrictVersion
 
+
```

Comment:
remove extra line

Reply:
it is proper pep8 to have two blank lines after imports